### ISSUE TITLE
Search: Incorrect tabbing order for the search page

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#119]
URL: https://hcdev.openplus.ca/en/search/site
User Agent: Chrome
Language: Both

Issue Description: When tabbing from the breadcrumb trail, the focus is sent to the right hand side filters. Users will have to tab through the filters and place their focus to the "Search" field which is located on the left. The tabbing order is from right to left which is not a logic tabbing order.
![image](https://user-images.githubusercontent.com/88776404/137157549-05c5ba2e-5984-4a79-a08d-ad73b5e58c09.png)

Steps to reproduce:

1. Navigate to the Search page https://hcdev.openplus.ca/en/search/site
2. Use the tab keys to move focus
3. The first tabbing focus is placed on the "Recall 2073" link under "Type" from the "Recalls and safety alerts" link in the breadcrumb trail

Expected results: The tabbing order should be from left to right. In this case, if users must navigate through the filters first, then the filters should be placed on the left.

**Comment by echouin** [comment]
@XiuJuanXia ﻿Would it be acceptable to add a link at the top of the facets that would display the following on focus "skip facets and go to results"?﻿﻿﻿﻿

**Comment by echouin** [comment]
@XiuJuanXia OP is asking if you have an answer for the previous question?

**Comment by XiuJuanXia** [comment]
@echouin after reviewing the code, adding a link at the top of the facets won't change the focus order.

The page is divided into two columns: searching results column and filters column. If turn off all the styles of the page, the filters column appears before the searching results column. Reviewing the code, class "pull-right" makes the filters column on the right and class="pull-left" makes the searching results column on the left.

Is it possible to switch the order of the two `<divs>`s for the two columns? Put the `<div>` for filters column after the `<div>` for searching results column.

**Comment by echouin** [comment]
@XiuJuanXia from OP: 
"Right now this structure is the as what is used on GOC topic pages. For example, this one from revenue Canada. https://www.canada.ca/en/services/taxes.html.

It is important to keep the focus order the way it is because you don't want people to have to tab through all the content to get to the facets.

if you move the facets to the left, the tab order matches visual order, but the page looks lousy which has a cognitive load impact as people try to figure out what is going on. My recommendation is to leave it alone."

**Eric comment: Based on this information, I would suggest marking as 'resolved'.**

**Comment by echouin** [comment]
@XiuJuanXia is this now verified/resolved?

**Comment by XiuJuanXia** [comment]
This structure is the same as [CDTS](https://cenw-wscoe.github.io/sgdc-cdts/docs/index-en.html) page. Closing this issue.
